### ISSUE TITLE
#region comments not working right since March update

### ISSUE THREAD

**Issue by josh-degraw** [Microsoft/vscode#47401]
Issue Type: <b>Bug</b>

Regions isn't working right anymore since I just barely updated. If i give a #region a name, only the #endregion directive will fold (below it). But if i just put //#region , with no text after, it works.

VS Code version: Code 1.22.1 (950b8b0d37a9b7061b6f0d291837ccc4015f5ecd, 2018-04-06T02:26:57.615Z)
OS version: Windows_NT x64 10.0.16299

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz (4 x 2904)|
|Memory (System)|15.88GB (7.47GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code\Code.exe|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (39)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-togglecodelens|cml|1.0.3
gitignore|cod|0.5.0
vscode-react-proptypes-intellisense|cun|0.0.2
vscode-markdownlint|Dav|0.14.1
jshint|dba|0.10.18
vscode-eslint|dba|1.4.8
vscode-npm-source|dku|1.0.0
githistory|don|0.4.0
gitlens|eam|8.2.0
tslint|eg2|1.0.28
vscode-npm-script|eg2|0.3.3
flow-for-vscode|flo|0.8.0
auto-close-tag|for|0.5.6
auto-rename-tag|for|0.0.15
code-runner|for|0.9.3
mdmath|goe|2.2.1
minify|Hoo|0.3.0
useful-react-snippets|igo|1.4.0
Ionide-fsharp|Ion|3.18.2
path-autocomplete|ion|1.9.0
latex-workshop|Jam|5.0.4
docthis|joe|0.6.0
vscode-codemetrics|kis|1.15.0
code-beautifier|mic|2.0.3
dotenv|mik|1.0.1
vscode-todo-parser|min|1.9.1
vscode-scss|mrm|0.6.2
csharp|ms-|1.14.0
vs-keybindings|ms-|0.2.0
debugger-for-chrome|msj|4.3.0
vscode-extension-auto-import|Nuc|1.4.1
vscode-versionlens|pfl|0.20.0
vscode-icons|rob|7.22.0
code-settings-sync|Sha|2.9.0
markdown-preview-enhanced|shd|0.3.4
autoimport|ste|1.5.3
latex-support|tor|1.5.1
vscode-todo-highlight|way|0.5.12
markdown-all-in-one|yzh|1.1.2

</details>

**Comment by NateMay** [comment]
I agree that this should be resolved as a bug. In the meantime you can revert back to  the previous folding configuration by adding the following to your User Settings:

```
    "[typescript]": {
       "editor.foldingStrategy": "indentation"
    },
```

**Comment by josh-degraw** [comment]
Looks like it works fine with just one region, but when I put two in there it got confused again

**Comment by josh-degraw** [comment]
Cool that fixed it. It may actually just be having a single-line comment of any kind right after that throws it off, which is what happened when I had multiple regions right after each other. But then it did it again when I just had a single `//` with nothing after it on the line following the `//#endregion`

**Comment by Emeryao** [comment]
@NateMay works around for me 👍

**Comment by mjbvz** [comment]
Root caused by https://github.com/Microsoft/TypeScript/issues/23270

Will be fixed by #47473
